deployments/ansible/roles/signalfx-agent/tasks/yum_repo.yml

```diff
@@ -0,0 +1,36 @@
+---
+- name: Add Signalfx Agent repo into source list
+  yum_repository:
+    name: signalfx-agent
+    description: SignalFx Agent Repository
+    baseurl: "{{ rhel_repo_url }}"
```

Comment:
Construct this variable here instead of in the vars file.

Reply:
done